frontends/ntt-pipeline/gen-ntt-pipeline.py

```diff
@@ -1,6 +1,12 @@
 import math
 import textwrap
 from prettytable import PrettyTable
+import sys
+from os.path import abspath, dirname, join
+
+# Access to futil_ast.
+sys.path.append(abspath(join(dirname(__file__), '../python')))
+from futil_ast import *
```

Comment:
Not something to fix here, but eventually we can make this an actually installable package (e.g., with [Flit](https://flit.readthedocs.io/en/latest/)) to avoid import path hackery.

Reply:
Can't disagree with that.